frontend/src/apis/group.ts

```diff
@@ -22,3 +24,21 @@ export const getCreatedGroupDataApi = async (dataForURL: DataForURL) => {
   const data = await response.json();
   return data;
 };
+
+// NOTE: 리뷰 목록 엔드포인트(gettingReviewList)에 요청을 보내고 있지만,
+// 요청 성격이 목록을 얻어오는 것이 아닌 유효한 groupAccessCode인지 확인하는 것이므로 group 파일에 작성함
+// 단, 해당 엔드포인트에 대한 정상 요청 핸들러가 동작한다면 아래 에러 핸들러는 동작하지 않음
+export const getIsValidGroupAccessCodeApi = async (groupAccessCode: string) => {
+  const response = await fetch(endPoint.gettingReviewList, {
+    method: 'GET',
+    headers: {
+      'Content-Type': 'application/json',
+      GroupAccessCode: groupAccessCode,
+    },
+  });
+
+  if (response.status === 400) throw new Error(INVALID_GROUP_ACCESS_CODE_MESSAGE);
+  if (response.status === 500) throw new Error(API_ERROR_MESSAGE.serverError);
```

Comment:
```suggestion
  if (response.status === 500) throw new Error(createApiErrorMessage(response.status));
```
위에처럼 createApiErrorMessage를 사용하지 않고 API_ERROR_MESSAGE.serverError를 바로 사용한 이유가 있을까요?

Reply:
이 요청에서는 특별히 400과 500 에러를 따로 처리하는 만큼 500 에러임을 코드에서 더 강조하려고 했습니다.
400은 따로 에러 메세지를 설정해주는데, 500을 처리할 때 `createApiErrorMessage`를 사용해서 status만 넘겨주면 뭐지? 싶을 것 같아서요!

그런데 지금 생각해보니까 특이 케이스인 400만 지금처럼 맨 처음에 따로 처리해주고 그 외의 에러들은 한꺼번에 처리해도 될 것 같다는 생각이 들었습니다.
그래서 400을 제외한 에러들은 `createApiErrorMessage`를 통해 한꺼번에 처리하도록 바꿨습니다~~